api/src/glfs-fops.c

```diff
@@ -1538,7 +1538,7 @@ glfs_pwritev_common(struct glfs_fd *glfd, const struct iovec *iovec, int iovcnt,
         ret = -1;
         errno = EINVAL;
         gf_smsg(THIS->name, GF_LOG_ERROR, errno, API_MSG_INVALID_ARG,
-                "size >= %llu is not allowed", GF_UNIT_GB, NULL);
+                "size >= ", "%llu is not allowed", GF_UNIT_GB, NULL);
```

Comment:
Why not use a more clean approach ? For example, the error could be `"Data size too large"`, and the additional info could be `"size = %llu"`

Reply:
Done